src/components/AppProvider/AppProvider.tsx

```diff
@@ -89,7 +89,7 @@ export class AppProvider extends Component<AppProviderProps, State> {
     const {theme = {}, children} = this.props;
 
     const {intl, link} = this.state;
-    const features = {newDesignLanguage: false, ...this.props.features};
+    const features = {...this.props.features};
```

Comment:
No need to spread and create a new object anymore. We can get rid of this variable and do the following below

```
<FeaturesContext.Provider value={this.props.features}>
```